### ISSUE TITLE
hpcrun should hide CS count when monitoring block time

### ISSUE THREAD

**Issue by jmellorcrummey** [HPCToolkit/hpctoolkit#238]
@laksono When measuring BLOCKTIME, CS should be a hidden metric rather than visible. It should also be renamed CNTXT_SWTCH.

**Comment by laksono** [comment]
I agree that CNTXT_SWTCH is more intuitive for general users although perf util shortens CONTEXT_SWITCHES into CS.
The change is tiny (2 lines). Should I create a pull request or just push it to the master branch?

```
diff --git a/src/tool/hpcrun/sample-sources/perf/kernel_blocking.c b/src/tool/hpcrun/sample-sources/perf/kernel_blocking.c
index ee2c9e0..6cb3f22 100644
--- a/src/tool/hpcrun/sample-sources/perf/kernel_blocking.c
+++ b/src/tool/hpcrun/sample-sources/perf/kernel_blocking.c
@@ -75,8 +75,9 @@
 // -----------------------------------------------------

 #define EVNAME_KERNEL_BLOCK     "BLOCKTIME"
-#define EVNAME_CONTEXT_SWITCHES "CS"
+#define EVNAME_CONTEXT_SWITCHES "CNTXT_SWTCH"

+#define HIDE_METRIC   0

 //******************************************************************************
 // forward declaration
@@ -235,6 +236,8 @@ register_blocking(kind_info_t *kb_kind, event_info_t *event_desc)
   event_desc->metric_desc =
     hpcrun_id2metric_linked(event_desc->hpcrun_metric_id);

+  event_desc->metric_desc->flags.fields.show = HIDE_METRIC;
+
   // ------------------------------------------
   // set context switch event description to be used when creating
   //  perf event of this type on each thread
```

**Comment by jmellorcrummey** [comment]
let’s use pull requests to modify master from now on.

**Comment by laksono** [comment]
Fixed in commit 2cf7ab8590ae34cead7f8c5f7d09cde3a3bce9d5
